storagenode/contact/service.go

```diff
@@ -26,7 +26,7 @@ type Config struct {
 	ExternalAddress string `user:"true" help:"the public address of the node, useful for nodes behind NAT" default:""`
 
 	// Chore config values
-	Interval time.Duration `help:"how frequently the node contact chore should run" releaseDefault:"1h" devDefault:"5s"`
+	Interval time.Duration `help:"how frequently the node contact chore should run" releaseDefault:"1h" devDefault:"30s"`
```

Comment:
Why 30s?

Reply:
arbitrary value, but leaves time for retries if the contact fails the first few times.

Reply:
updated it back to 5s to make it easier to debug in storj sim